user-interface/src/review-orders/TransferOrderAccordion.tsx

```diff
@@ -367,3 +549,157 @@ function CaseSelection(props: CaseSelectionProps) {
     </>
   );
 }
+
+interface ConfirmationModalProps {
```

Comment:
We should probably extract the Confirmation Modal into its own file

Reply:
I'm okay with it being here for now, but yeah, we'll almost certainly end up with this being extracted at some point in the future.